mlx/traceability.py

```diff
@@ -330,20 +339,22 @@ def _parse_description(description, attr_values):
     Args:
         description (str): Description of the merge/pull request.
         attr_values (list): List of the two possible attribute values (str).
+        merge_request_id (int): Merge/Pull request ID.
 
     Returns:
-        (dict) Dictionary with key-value pairs with item IDs (str) as keys and the attribute values (str) as values.
+        (dict) Dictionary with key-value pairs with item IDs (str) as keys and ItemInfo (attr_val, mr_id) (namedtuple)
+            as values.
     """
     query_results = {}
     for line in description.split('\n'):
         # catch the content of checkbox and the item ID after the checkbox
-        match = search(r"^\s*[\*-]\s+\[(?P<checkbox>[\sx])\]\s+(?P<target_id>[\w\-]+)", line)
-        if match:
-            if match.group('checkbox') == 'x':
-                attr_value = attr_values[0]
+        cli_match = search(r"^\s*[\*-]\s+\[(?P<checkbox>[\sx])\]\s+(?P<target_id>[\w\-]+)", line)
```

Comment:
can we ensure target_id regexes to have the same regex (also above?) That might avoid some problems in future.

Reply:
Earlier today you didn't want this. We are fine with this particular regex, but other users might format their item IDs differently. I can make it configurable in conf.py, but where?

Reply:
This was more consistency comment than the regex dependency. Nice idea to put it int conf.py probably next to `merge_request_id` we can do something with `checkbox-item-regexp`.

Reply:
done: d24c7ef1cba27f38cfe5c00d551ddefe2d5a27ca